packages/adapter/src/methods.ts

```diff
@@ -70,6 +70,16 @@ export async function exportSeed(this: MetamaskPolkadotSnap): Promise<string> {
   return (await sendSnapMethod({ method: 'exportSeed' }, this.snapId)) as string;
 }
 
+export async function exportAccount(
+  this: MetamaskPolkadotSnap,
+  jsonPassphrase?: string
```

Comment:
Remove `jsonPassphrase` there is no usage of it in snap

Reply:
Left it optional because polkadot js extension must have a passphrase to successfully import account.
Leaving it as empty does not allow restoring.
![Screenshot from 2024-06-18 13-20-33](https://github.com/ChainSafe/metamask-snap-polkadot/assets/51483483/e2903416-7d0e-4706-b59b-b50e6040b64b)

Reply:
but like reviewing [exportAccount.test.ts](https://github.com/ChainSafe/metamask-snap-polkadot/pull/234/files#diff-f97175f9a56050187928b6b7ae318c79a78b28fecb099564382fed8baf5012e8) seems like there is no point of using it

If the extension uses a password field that is not used anywhere (bad U, itX) should not affect our API

Reply:
Why we wouldn't allow this option, we lose nothing extending the already existing 
`KeyringPair.toJson(passphrase?: string)` method.